flink-python/pyflink/datastream/connectors/__init__.py

```diff
@@ -73,6 +73,13 @@ def _install():
     setattr(connectors, 'StreamFormat', file_system.StreamFormat)
     setattr(connectors, 'StreamingFileSink', file_system.StreamingFileSink)
 
+    # pubsub
+    from pyflink.datastream.connectors import pubsub
```

Comment:
This is for backward compatibility. Since pubsub is newly introduced, so this is not necessary and could be removed.

Besides, we may need to add some documentation here: https://github.com/apache/flink/blob/master/flink-python/pyflink/datastream/__init__.py#L209